src/AudioUnit/AudioUnit.cs

```diff
@@ -1272,12 +1272,15 @@ enum AudioUnitPropertyIDType { // UInt32 AudioUnitPropertyID
 		ParameterHistoryInfo = 53,
 		Nickname = 54,
 		OfflineRender = 37,
-		[Availability (Introduced = Platform.iOS_8_0 |Platform.Mac_10_3)]
+		[Availability (Introduced = Platform.iOS_8_0)]
+		[Availability (Introduced = Platform.Mac_10_3)]
```

Comment:
Do we even NEED the 10_3 tags?  Don't we only support from like 10.7+ anyway?  Is it worth just dropping these if we're already going to have diffs here?